UnityProject/Assets/Scripts/UI/Systems/Lobby/CharacterCustomization.cs

```diff
@@ -858,7 +892,7 @@ public void SetDropDownBody(ObjectList GameObjectBody)
 			}
 		}
 
-		public void SubSetBodyPart(BodyPart bodyPart, string path)
+		private void SubSetBodyPart([NotNull] BodyPart bodyPart, [NotNull] string path)
```

Comment:
the ``NotNull`` attribute means that this can't be null so there is no need to null check it inside the method body, isn't that the case? If so, you should remove the attribute.

Reply:
Rider added this for me, going to change it